### DISCUSSION TITLE
Docusaurus配置Gitalk评论插件 - 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#195] [General]
# Docusaurus配置Gitalk评论插件 - 愧怍的小站

之前使用 vuepress 的时候，使用的评论系统是Valine，可是匿名用户也能直接评论，虽说会过滤垃圾信息，但是后台查看评论与通知总感觉没有那么实在。

https://kuizuo.cn/docusaurus-gitalk-plugin/#comments

**Comment by qingfengfenga** [comment]
你好，我目前参考你的这个文章 `https://kuizuo.cn/docs/docusaurus-comment#giscus` 添加的评论系统，好像有点问题，可以麻烦你帮我看一下吗，目前提示`giscus.app`拒绝连接 。

评论系统显示异常的博客地址 `https://yanpengwang.com/blog/hosting-docusaurs-pages-using-github-pages`
